site/docs/master/restore-reference.md

```diff
@@ -9,6 +9,44 @@ velero restore create RESTORE_NAME \
   --from-backup BACKUP_NAME \
   --namespace-mappings old-ns-1:new-ns-1,old-ns-2:new-ns-2
 ```
+## What happens when user removes restore Objects. 
+A **restore** object represents a restore operation.There are two types of deletion for restore objects:
+### 1. Deleting with **`velero restore delete`**
+This command will delete the custom resource representing it, along with its individual log and results files.But, it will not delete any objects that were created by it from your cluster.
+### 2. Deleting with **`kubectl -n velero delete restore`**
+This command will delete the custom resource representing the restore, but will not delete log/results files from object storage, or any objects that were created by the restore in your cluster.
+
+## Restore command-line options
```

Comment:
I think I'd rather not show the actual output of these commands in the documentation, as it's likely to get out-of-date as we add new commands/options.  Let's simplify this section somewhat - I think we can simply say something like (feel free to adjust wording): 

To see all commands for restores, run `velero restore --help`.  

To see all options associated with a specific command, provide the `--help` flag to that command. For example, `velero restore create --help` shows all options associated with the `create` command.